client/src/frontend/components/resource-slide/SlideImage.js

```diff
@@ -1,38 +1,28 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import ResourcePreview from "frontend/components/resource-preview";
-import IconComposer from "global/components/utility/IconComposer";
+import get from "lodash/get";
+import { Zoom } from "./Slide";
 
 export default class ResourceListSlideFigureImage extends Component {
-  static propTypes = {
-    resource: PropTypes.object,
-    enableZoom: PropTypes.bool
-  };
+  static displayName = "ResourceList.Slide.Image";
 
-  static defaultProps = {
-    enableZoom: true
+  static propTypes = {
+    resource: PropTypes.object.isRequired
   };
 
   render() {
-    const attr = this.props.resource.attributes;
+    const background = get(
```

Comment:
Prefer get methods on the class instance to declaring constants in the render method, IMO.